docs/mm_bot/architecture.md

```diff
@@ -36,30 +40,37 @@ The bot architecture is as follows:
 
 ![architecture.png](images/architecture.png)
 
-The bot is composed of the following components:
-- **Main Process**: The main process of the bot. It has the following subcomponents:
-    - `Main Order Indexer`: The `Main Order Indexer` is responsible for indexing the orders from 
-    the pending blocks.
-  - `Order Processor`: Responsible for processing the orders.
-  - `Failed Orders Processor`: Responsible for retrying the failed orders.
-  It runs every 5 minutes.
-  - `Accepted Blocks Processor`: Responsible for indexing the orders that belong to accepted blocks. It runs every 5 minutes.
-- **Database**: The database is used to store the following data:
-  - Orders
-  - Errors
-  - Block numbers
-
-An important aspect of the bot is that it must be able to handle multiple orders concurrently.
+For processing the orders from a specific L2 is needed an `Orders Processor` for each L2.
+Each `Orders Processor` has an `Orders Indexer` and an `Order Executor`.
+
+The `Orders Indexer` is responsible for indexing the orders from the L2 Escrow. It indexes orders reading the events, 
+and when it finds a new order, it stores it in the database. Then, `Orders Processor` is able to assign the new order 
+to an `Order Executor`.
+
+The `Order Executor` is responsible for processing and individual order. It means, it will transfer the funds to the 
```

Comment:
Typo and -> an

```suggestion
The `Order Executor` is responsible for processing an individual order. It means, it will transfer the funds to the 
```